be/src/olap/olap_snapshot_converter.h

```diff
@@ -47,7 +47,7 @@ class OlapSnapshotConverter {
     // convert olap header to tablet meta pb, convert delta to rowsetmetapb
     // pending delta is not in tablet meta any more, so that convert pending delta to rowset and add it to pending rowsets
     // as a return value
-    OLAPStatus to_tablet_meta_pb(const OLAPHeaderMessage& olap_header, TabletMetaPB* tablet_meta_pb, 
+    OLAPStatus to_tablet_meta_pb(OLAPHeaderMessage* olap_header, TabletMetaPB* tablet_meta_pb,
```

Comment:
Originally, this function only converts the header to meta.
Can you implement another function to finish new added operation to make it less complicate?